FBAllocationTracker/NSObject+FBAllocationTracker.mm

```diff
@@ -46,3 +48,26 @@ - (void)fb_newDealloc
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
+
+
+bool fb_isZombieObject(_Nullable id object) {
+  if (object == nil) { return true; }
+  
+  const char *className = object_getClassName(object);
+  static const char *zombiePrefix = "_NSZombie";
+  return strncmp(className, zombiePrefix, strlen(zombiePrefix)) == 0;
```

Comment:
By changing `zombiePrefix` from a pointer to an array, this can then use `sizeof()` instead of `strlen()`, to make the string length calculation done once at compile time.